test/framework/vmoperator/vmoperator.go

```diff
@@ -392,36 +392,6 @@ func ReconcileDependencies(ctx context.Context, c client.Client, dependenciesCon
 		if retryError != nil {
 			return retryError
 		}
-
```

Comment:
(if you know) why is this not needed anymore?

Reply:
Should we update the comment in l.363?

Reply:
CR is gone in v1alpha2 and no code of vm-operator uses it anymore, also no conversion to a different type.